### ISSUE TITLE
Sonatype maven repo lacks (dependencies for) HLRC 7.11.0

### ISSUE THREAD

**Issue by spinscale** [elastic/elasticsearch#68889]
**Elasticsearch version** (`bin/elasticsearch --version`): 7.11.0

**Steps to reproduce**:

Creating  an empty gradle project, and adding the HLRC results in this error

 1. Create a new gradle project
 2. Add `implementation 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.11.0'` to your dependencies
 3. Run `./gradlew clean check`

**Provide logs (if relevant)**:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lib:test'.
> Could not resolve all files for configuration ':lib:testRuntimeClasspath'.
   > Could not find org.elasticsearch:elasticsearch-plugin-classloader:7.11.0.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/elasticsearch/elasticsearch-plugin-classloader/7.11.0/elasticsearch-plugin-classloader-7.11.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :lib > org.elasticsearch.client:elasticsearch-rest-high-level-client:7.11.0 > org.elasticsearch:elasticsearch:7.11.0
```

Also, the rest high level client do not seem to be on maven central for 7.11.0 and 6.8.14, see https://search.maven.org/artifact/org.elasticsearch.client/elasticsearch-rest-high-level-client

**Comment by mark-vieira** [comment]
The other issue here is that `elasticsearch-plugin-classloader` aritfact is missing, which will cause issue. I don't believe this is actually needed at runtime though so you should be able to safely exclude that dependency for now and press forward until we get this fixed for 7.11.1.

**Comment by mark-vieira** [comment]
This has been fixed in https://github.com/elastic/elasticsearch/pull/68926 and should be resolved in the 7.11.1 release.

**Comment by spinscale** [comment]
The same issue also arises for external plugins which depend on `org.elasticsearch.gradle:build-tools`

What would be the correct step to exclude the elasticsearch-plugin-classloader dependency, as I do not mention any elasticsearch dependency in the `dependencies` section. This does not work

```
compile("org.elasticsearch:elasticsearch:7.11.0") {
  exclude 'org.elasticsearch:elasticsearch-plugin-classloader:7.11.0'
}
```

**Comment by mark-vieira** [comment]
Yes, the runtime dependency in the test framework is actually deliberate and the initial reason for the addition of this dependency to begin with so there's no simple workaround for getting past that. The 7.11.1 release should be this week so that will sort this out.

**Comment by davemoore-** [comment]
I see elasticsearch-plugin-classloader [on Maven](https://repo.maven.apache.org/maven2/org/elasticsearch/elasticsearch-plugin-classloader/) now that 7.11.1 is out, and I'm able to build projects that require it.

Will this be available on Maven for 7.11.0 so we can avoid a version-specific [workaround](https://github.com/elastic/elasticsearch/issues/68888#issuecomment-779489027) in our build matrix?

**Comment by spinscale** [comment]
just tried with 7.11.1 and everything worked smoothly for my plugins!

**Comment by mark-vieira** [comment]
> Will this be available on Maven for 7.11.0 so we can avoid a version-specific workaround in our build matrix?

We do not intend to retroactively publish the artifact for 7.11.0. That version is effectively end-of-life now that 7.11.1 is out.
